doc/source/downloading_cesm.rst

```diff
@@ -1,7 +1,7 @@
 .. _downloading:
 
 ============================
-Downloading CESM (|version|)
+Downloading CESM2 (|version|)
```

Comment:
This one will show up like "Downloading CESM2 (CESM2.2)". That kind of makes the "2" in "CESM2" feel redundant, but I don't feel strongly.